app/experimenter/experiments/views.py

```diff
@@ -100,11 +143,75 @@ class ExperimentFilterset(filters.FilterSet):
         label="Show archived experiments", widget=forms.CheckboxInput()
     )
 
+    show_experiment_changes = filters.ChoiceFilter(
```

Comment:
Nit: a better name might be `experiment_date_field`

Reply:
Made this change!